portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +26,26 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  ArrayList<String> messageArrayList = new ArrayList<String>();
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Gracie!</h1>");
+    response.setContentType("application/json");
+    Gson gson = new Gson();
+    String messageJson = gson.toJson(messageArrayList);
+    response.getWriter().println(messageJson);
+  }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String comment = request.getParameter("comment-input");
+    if (comment == null) {
+        comment = "There was nothing inputted in the text area.";
+    }
```

Comment:
This will add the comment to the array list. Shall we instead ignore the post request if it contains an empty comment?

Reply:
Adjusted if statement to ignore empty comments.